lib/rails_event_store/event_repository.rb

```diff
@@ -0,0 +1,11 @@
+module RailsEventStore
```

Comment:
Hmm I don't like this approach. It will require us to add here new code each time someone will implement new event repository. I would just remove it if we will go that way (not having RESAR in RES dependencies).